e2e/assets/gzip/dfx.json

```
@@ -0,0 +1,16 @@
+{
+    "version": 1,
+    "canisters": {
+        "gzipped": {
+            "type": "custom",
+            "candid": "main.did",
+            "wasm": "main.wasm.gz",
+            "build": "sh -c 'gzip -c main.wasm >main.wasm.gz'"
+        }
+    },
+    "networks": {
+        "local": {
+            "bind": "127.0.0.1:8000"
+        }
+    }
```

Tests should use the shared local network, unless they are specifically testing project-specific local networks
```suggestion
    }
```

---

Oh, I wasn't sure what that meant.